src/Components/Tasks/Tasks.tsx

```diff
@@ -342,403 +431,403 @@ const Tasks: React.FC = () => {
-                    {tasks.map((task: any) => (
-                      <tr key={task?.id}>
-                        <th scope="row">{task?.title}</th>
-
-                        <td className=' text-white' style={{ textAlign: 'center' }}>
-                          <div style={{
-                            backgroundColor:
-                              task?.status === 'ToDo'
-                                ? '#E4E1F5'
-                                : task?.status === 'InProgress'
-                                  ? '#EF9B28'
-                                  : task?.status === 'Done'
-                                    ? '#009247'
-                                    : 'inherit',
-                            borderRadius: '15px',
-                            fontSize: '16px',
-                            padding: '10px', // Adjust the padding as needed
-                            fontWeight: '250',
-                            fontFamily: 'Montserrat-Regular',
-                          }}>
-                            {task?.status}
-                          </div>
-                        </td>
-                        <td>{task?.description}</td>
-                        <td>{task.employee.userName}</td>
-                        <td>{task.project.title}</td>
-                        <td>{new Date(task.creationDate).toLocaleDateString()}</td>
-                        <td>
-                          <button onClick={() => showViewModal(task?.id)}
-                            className="p-0 border-0 bg-white">
-                            <i
-
-                              className="fa fa-eye  text-info px-1"
-                            ></i>
-                          </button>
-                          <button className="p-0 border-0 bg-white">
-                            <i
-                              onClick={() => showUpdateModal(task)}
-                              className="fa fa-pen  text-warning px-1"
-                            ></i>
-                          </button >
-                          <button onClick={() => showDeleteModal(task.id)}
-                            className="p-0 border-0 bg-white">
-                            <i
-
-                              className="fa fa-trash  text-danger px-1"
-                            ></i>
-                          </button>
-
+        {
+          userRole === 'Manager' ? (
+            <div className="table-container1 vh-100">
+              <div className="w-50">
+                <Select
+                  options={projectList.map((project) => ({ value: project.id, label: project.title }))}
+                  value={
+                    selectedProjectId
+                      ? { value: selectedProjectId, label: projectList.find((project) => project.id === selectedProjectId)?.title || '' }
+                      : null
+                  }
+                  onChange={(selectedOption) => handleProjectSelection(selectedOption?.value)}
```

Comment:
The onChange handler when given a callback function it take the event not the selectedOption, and that is why you are not getting any value while searching.
To fix this issue you convert it as follows:
`onChange={(event) => handleProjectSelection(event?.target?.value)}`

Reply:
My code working when i do => onChange={(event) => handleProjectSelection(event?.target?.value)} not work